app/features/hit_fox_feed.rb

```diff
@@ -0,0 +1,23 @@
+require 'entities/github_feed'
+require 'jbuilder'
```

Comment:
why is this in app/features? It should be in models or in the lib folder. Also, isn't jbuilder loaded when the app is initialized?